src/store/dashboard/sagas/transactions.sagas.ts

```diff
@@ -0,0 +1,45 @@
+import { put, call, select } from 'typed-redux-saga';
+import { createWatcherSaga } from '../../utils/utils.sagas';
+import { subgraphClientSelector } from '../../app/app.selectors';
+import { transactionsQuery } from '../../../queries/transactionsQuery';
+import { dashboardActions } from '../dashboard.slice';
+import { mockedAccount as account } from './transactions.mocks';
+
+export function* fetchTransactions() {
+  try {
+    const subgraphClient = yield* select(subgraphClientSelector);
+
+    /* TODO: for now we are using mocked account address,
+    that alrady have some tx on testnet,
+    but in prod we should use selector to grab current wallet address 
+
+    const account = yield* select(accountSelector);
```

Comment:
I wouldn't commit these changes with this mock, it's a nice way to test if transactions are loaded correctly, but for me it would be better to have the version with proper account even though for now we don't have any items on them.